crates/hc_sandbox/src/run.rs

```diff
@@ -68,16 +68,16 @@ pub async fn run(
         launch_info.app_ports.push(port);
     }
 
+    crate::save::lock_live(std::env::current_dir()?, &sandbox_path, admin_port).await?;
```

Comment:
It's problematic to declare the sandbox running before creating this file. It means that commands run against the sandbox that expect to find this `.hc_live_x` file will not find a running conductor. I don't think the tests are relying on this but it being in the wrong order is probably partly why.

Reply:
makes sense to me, yes.